homeassistant/components/cloud/http_api.py

```diff
@@ -592,3 +593,15 @@ def _account_data(cloud):
         connection.send_result(msg["id"])
     else:
         connection.send_error(msg["id"], ws_const.ERR_UNKNOWN_ERROR, "Unknown error")
+
+
+@websocket_api.async_response
+@websocket_api.websocket_command({"type": "cloud/thingtalk/convert", "query": str})
+async def thingtalk_convert(hass, connection, msg):
+    """Convert a query."""
+    cloud = hass.data[DOMAIN]
+
+    with async_timeout.timeout(10):
+        connection.send_result(
+            msg["id"], await thingtalk.async_convert(cloud, msg["query"])
```

Comment:
Can we forward the error message? Should also remove `raise_on_error` in `hass_nabucasa`?

```
        result = await thingtalk.async_convert(cloud, msg["query"])
        error = result.get("error")
        if not error:
            connection.send_result(msg["id"], result)
        else:
            connection.send_error(msg["id"], ws_const.ERR_UNKNOWN_ERROR, error)
```

Reply:
Fixed.